pkg/http/server.go

```diff
@@ -112,10 +114,19 @@ func (s *server) useProxy(next fasthttp.RequestHandler) fasthttp.RequestHandler
 				ctx.RemoteAddr(),
 				ctx.Host(),
 				proto))
-		// Add X-Forwarded-For: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For
-		ctx.Request.Header.Add("X-Forwarded-For", ctx.RemoteAddr().String())
-		// Add X-Forwarded-Host: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host
-		ctx.Request.Header.Add("X-Forwarded-Host", fmt.Sprintf("%s", ctx.Host()))
+
+		hostAddress, err := utils.GetHostAddress()
```

Comment:
I would consider getting the host address and hostname once outside the actual middleware and then just adding the headers in the middleware.